Scout.py

```diff
@@ -2,12 +2,15 @@
 # -*- coding: utf-8 -*-
 
 import sys
+import asyncio
```

Comment:
This is not compatible with python 2, no?

Reply:
It indeed seems like it's not, (it was introduced with 3.5). I think there's a way to make it compatible though, I'll look into it

Reply:
I'm not really against making the tool only compatible with 3.x, but there would have to be some significant benefits to removing compatibility with 2.7